ipmof_energymap.py

```diff
@@ -20,7 +20,9 @@
     mof_list = get_mof_list(mof_path_list, force_field)
 else:
     # Create MOF list by reading structure files from a directory
-    mof_list = get_mof_file_list(sim_dir['mof_dir'], 'cif', force_field)
+    mof_path_list = os.listdir(sim_dir['mof_dir'])
+    mof_path_list = [os.path.join(sim_dir['mof_dir'], path) for path in mof_path_list]
+    mof_list = get_mof_list(mof_path_list, force_field)
```

Comment:
You should know about globs! Might not be helpful, but you should still know about them!